msyncd/gschemas/com.meego.msyncd.gschema.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schemalist>
+  <schema id="com.meego.msyncd" path="/com/meego/msyncd/">
+    <key name="allow-scheduled-sync-over-gsm" type="b">
+      <summary>Scheudled Sync over GSM</summary>
```

Comment:
Hey sorry I did not realize that QGSettings was a Ubuntu project (https://code.launchpad.net/~system-settings-touch/gsettings-qt/trunk) very similar to MGConfItem.

Probably the best here is use native gconf glib API, to avoid  extra dep.. What do you think?